architecture-decision-records/0035-metadata-csv-validation.md

```diff
@@ -0,0 +1,31 @@
+# 35. Metadata CSV Validation
+
+**Date**: 2024-10-07
+
+## Context
+
+Digital assets sent to The National Archives (TNA) are transferred with associated metadata in CSV format. The entry point for digital assets to TNA will be through the services provided by the Digital Selection and Transfer team.
+
+The structure and integrity of the CSV file is checked before the actual [data is validated](0034-metadata-validation.md) 
+
+## Structure and integrity checks
+
+1. **Virus check**: All files uploaded to TDR are checked for viruses using the virus check lambda
+2. **[UTF-8](utf-8-validation)**: The CSV file must be UTF-8 format
+3. **[Valid CSV file](valid-csv-file)**: Is it a CSV file
+4. **[Required columns](required-columns)**: Certain columns are required (such as closure information columns) for all transfers
+
+### UTF-8 Validation 
```

Comment:
Can you expand on the alternatives to using the BOM. Highlighting the advantages / disadvantages. 

Can you give reasons why the BOM was selected as the preferred method. Will need to cover the limitations with MACs / Libre office.